### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted _YourFirstExtension_Zoom: At zoom level 200%, the label on the button is not visible, no tool tip is provided

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2791]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS1.1.1, MAS1.4.4

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension) OR
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "API" link.
3.From left navigation, select "Get Started".
4.Navigate to "Your First Extension" link.
5.Go to browser settings and increase the Zoom to 200%.
6.Observe the changes that are applicable on the button "Edit" 

**Actual:**
Scenario: When zoom level for the page is increased to Zoom 200%:
The label for the button "Edit" is not present.
No tool tip or other alternative is provided so that the user can identify the button as edit button.

**Expected:**
Scenario: When zoom level for the page is increased to Zoom 200%:
The label for the button "Edit" must be present.
Tool tip or other alternative assistance must be  provided so that the user can identify the button as edit button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Keyboard only users will be not able to identify the button if the label is missing or if any other alternative is not provided such as tool tip.

People who have difficulty perceiving visual content. Assistive technology can read text aloud, present it visually, or convert it to braille.
Text alternatives or tooltips may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc.
People who are deaf, are hard of hearing, or who are having trouble understanding audio information for any reason can read the text presentation. Research is ongoing regarding automatic translation of text into sign language.
People who are deaf-blind can read the text in braille.
Additionally, text alternatives support the ability to search for non-text content and to repurpose content in a variety of ways.

**MAS Reference:**
MAS 1.4.4 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d})

MAS1.1.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

**Attachment for Reference:**
![A11y_code visualstudio com_APIGetStarted _YourFirstExtension_Zoom](https://user-images.githubusercontent.com/51443731/60000336-9cf8a800-9681-11e9-9f82-39a93ef2ce5e.png)
![A11y_code visualstudio com_APIGetStarted _YourFirstExtension_Zoom_1](https://user-images.githubusercontent.com/51443731/60000337-9cf8a800-9681-11e9-9b46-9a4867bf4530.png)

**Comment by octref** [comment]
Fixed as part of #2784.

**Comment by bhagwatshelke1047** [comment]
@octref , @gregvanl This issue is still reproducing can you please open the issue.

**Comment by gregvanl** [comment]
Fixed, tool tip added.

![image](https://user-images.githubusercontent.com/12818376/82932598-887df900-9f3d-11ea-9a2e-560df90ad5c3.png)
